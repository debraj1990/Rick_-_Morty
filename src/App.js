import React from 'react';
import './App.css';
import Layout from './containers/Layout/Layout'

function App() {
  return (
    <div>
        <Layout></Layout>
      </div>
  );
}

export default App;
