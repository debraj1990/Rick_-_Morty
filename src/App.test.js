import React from "react";
import { shallow } from 'enzyme';
import App from "./App";


const wrapper = shallow(<App />);

describe('App component', () => {
  it('testing the root app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper.getElements()).toMatchSnapshot();
  });
});



