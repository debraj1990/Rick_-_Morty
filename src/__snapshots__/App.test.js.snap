// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component testing the root app 1`] = `
Array [
  <div>
    <UNDEFINED />
  </div>,
]
`;
